### PR TITLE
add: openshift example values

### PR DESCRIPTION
**Pull request by codekow**
### Description of the change

Add chart values to examples for deployment on OpenShift

### Benefits

The ability to deploy label studio on OpenShift easily with `helm`

### Possible drawbacks

Unknown

### Applicable issues

NA

### Additional information

OpenShift routes are a simplified ingress configuration - chart does not need modification at this time

### Checklist

N/A - adding example values, not changing chart

### CODE REVIEW

File: heartex/label-studio/example-values/label-studio-on-openshift.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Label Studio running on OpenShift
+
+app:
+  podSecurityContext:
```
Comment (farioas) [inline heartex/label-studio/example-values/label-studio-on-openshift.yaml:4]: Why it miss?
```
app:
  rbac:
    create: true
rqworker:
  rbac:
    create: true
```

File: heartex/label-studio/example-values/label-studio-on-openshift.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    enabled: false
+  containerSecurityContext:
+    enabled: false
+  # ingress:
+  #   enabled: true
+
+rqworker:
```
Comment (farioas) [inline heartex/label-studio/example-values/label-studio-on-openshift.yaml:11]: rqworker is not required for LabelStudio OpenSource

File: heartex/label-studio/example-values/label-studio-on-openshift.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+postgresql:
+  enabled: true
+  primary:
+    podSecurityContext:
+      enabled: false
+    containerSecurityContext:
+      enabled: false
+
+redis:
```
Comment (farioas) [inline heartex/label-studio/example-values/label-studio-on-openshift.yaml:25]: rqworker is not required for LabelStudio OpenSource

**Review by farioas — CHANGES_REQUESTED** [review]

File: heartex/label-studio/example-values/label-studio-on-openshift.yaml (hunk 3 above)
Comment (codekow) [reply heartex/label-studio/example-values/label-studio-on-openshift.yaml:25]: If someone is installing LabelStudio enterprise could they use rqworker? The example is not exclusive.

Comment (codekow) [reply heartex/label-studio/example-values/label-studio-on-openshift.yaml:21]: Have updated the notes in the values

Comment (farioas) [reply heartex/label-studio/example-values/label-studio-on-openshift.yaml:31]: I suggest to split this example into two: `label-studio-on-openshift.yaml` and `label-studio-enterprise-on-openshift.yaml`

Comment (codekow) [reply heartex/label-studio/example-values/label-studio-on-openshift.yaml:31]: @farioas 

I do not have a license to test the enterprise version with. I am removing the comments and will only focus on a community deployment

**Review by farioas — APPROVED** [review]

**Pull request merged**